allure-pytest/test/labels/severity/function_severity_test.py

```diff
@@ -13,3 +13,17 @@ def test_decorated_function():
 
     """
     pass
+
+
+@pytest.allure.CRITICAL
```

Comment:
1) Because it is constant value
2) I try to make migrate to pytest-allure2 easy and it is previous version notation